src/cursor-doc/model.ts

```diff
@@ -9,6 +9,17 @@ export function initScanner(maxLength: number) {
   scanner = new Scanner(maxLength);
 }
 
+/**
+ * @param text The text to look at for the line ending seq.
+ * @returns The first line ending sequence found in TEXT, if any.
+ */
+export function getFirstLineEndingSeq(text: string) {
+  const found = text.match(/\r\n|\n/g);
+  if (found) {
+    return found[0];
+  }
```

Comment:
I think it can be written as:
```suggestion
  return text.match(/\r\n|\n/g)?[0];
```
(I haven't tested 😄 )

Reply:
Updated, it appears it requires `?` to be followed by `.`